mirrorlist/mirrorlist_client.wsgi

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
This seems problematic as we still deploy on python2 systems.

Reply:
Ok. It was a result of this error I was getting at `fedpkg mockbuild`:
```
*** ERROR: ambiguous python shebang in /usr/share/mirrormanager2/mirrorlist_client.wsgi: #!/usr/bin/env python. Change it to python3 (or python2) explicitly.
```
Changed it to `#!/usr/bin/env/python2` and it's passing.

Reply:
That is also incorrect. It should be python2 on Python 2 system and python3 on other systems.

Reply:
Is the shebang really necessary here? Python2 is unsupported.

Reply:
Shebang found redundant and removed.

Reply:
Good idea.